modules/installation-configuration-parameters.adoc

```diff
@@ -788,6 +789,27 @@ ifdef::rhv[]
 |`platform.ovirt.ingress_vip`
 |Required. An IP address on the machine network that will be assigned to the Ingress virtual IP (VIP).
 |String. Example: `10.46.8.232`
+
+|`platform.ovirt.affinityGroups`
+|Required. A list of affinity groups that should be created during the installation process.
+|List of objects.
+
+|`platform.ovirt.affinityGroups.description`
+|Required. A description of the affinity group.
```

Comment:
all the subfields are required only if the affinityGroups is provided, not sure if we need to change anything because of that

Reply:
@stoobie FYI

Reply:
@Gal-Zaidman I changed all subfields to `Required if you include `platform.ovirt.affinityGroups`.`